compiler/damlc/tests/daml-test-files/EmptyContractKeyMaintainers.daml

```diff
@@ -0,0 +1,19 @@
+-- @ERROR Attempt to create a contract key with an empty set of maintainers
+module EmptyContractKeyMaintainers where
+
+import qualified DA.Next.Set as Set
+import DA.Next.Set (Set)
+
+template NoMaintainer
+  with
+    sig: Party
+  where
+    signatory sig
+    key sig : Party
+    maintainer (Set.empty : Set Party)
```

Comment:
We can use lists of parties here as well. That would make it simpler.
```suggestion
    maintainer []
```

Reply:
done